src/Domain/PostalAddress.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace MatchBot\Domain;
+
+use Doctrine\ORM\Mapping\Column;
+use Doctrine\ORM\Mapping\Embeddable;
+use MatchBot\Application\Assertion;
+
+/**
+ * Value object for convenience and organisation but with very minimal validation since data is
+ * currently all entered via Salesforce so we can't throw back to the UI for any bad data.
+ */
+#[Embeddable]
+readonly class PostalAddress
+{
+    public static function of(
+        string $line1,
+        ?string $line2,
+        ?string $city,
+        ?string $postalCode,
+        ?string $country,
+    ): self {
+        return new self($line1, $line2, $city, $postalCode, $country);
+    }
+
+    private function __construct(
+        #[Column(nullable: true)]
```

Comment:
took me a while trying to work out why doctrine seemed to be ignoring the `Embedded` annotation on the Charity until I realised I also had to annotate each property here.